docs/index.adoc

```diff
@@ -37,7 +37,14 @@ include::installing-korrel8r-ocp.adoc[leveloffset=+1]
 include::using-korrel8r.adoc[leveloffset=+1]
 
 [id="index-api-reference"]
-== API reference documentation
```

Comment:
Remove the ID also if you're removing the heading, or add it to the included .adoc if possible

Reply:
+1. Is there a recommended practice about top-level headings being inside/outside included files? Inside makes the include more complete, but outside lets you make top level headings consistent. E.g. the REST API used to be titled "Korrel8r REST API" which made sense when it was a separate doc, but when it's included in the Korrel8r book it's seems like stutter to have chapters called "Korrle8r this", "Korrel8r that"...